src/templates/interest/html.mjml

```diff
@@ -0,0 +1,78 @@
+
+          <p>For those of you who have never done a confidence pool before, they are very easy to do. The objective is to
+            pick the winner of each game for the current week. When making your selections, you will assign a point value,
+            or "confidence" value, based on how sure you are of each pick. 1 means it could go either way and 16 means it's
+            a sure thing. If you are right, you win that amount of points and the player with the highest point total wins. Payouts are done to
+            the winner(s) each week and top 3 places at the end for the overall score. More general details about confidence pools
+            can be found here:</p>
+
+          <p><a href="http://ezinearticles.com/?How-to-Play-Fantasy-Football-Confidence-Pools&id=2596977">How to Play NFL Confidence Pools</a></p>
+
+          <p class="subtitle">Registering</p>
+
+          <p>If you have played previously, you can click the button at the bottom of this email and sign in to complete registration for this upcoming season.</p>
+
+          <p>If you are a new player, welcome! You will also need to click the button below, however, you will need to register. You have
+            the option of registering with an email and password, or with a Twitter or Google account. You also have the
+            option of registering with email and then linking your Twitter and/or Google account, allowing you one-click
+            sign-ins with all three methods. This is what we recommend, though certainly not required.</p>
+
+          <p>Once you are fully registered, you will be able to sign in and start making picks. The pool contains full
+            information on how to make your picks and play, however, if you have any questions, please do not hesitate to contact
+            us at this email address.</p>
+
+          <p class="subtitle">Survivor Pool</p>
+          <p>There is also an optional mini-game survivor pool. If you've never taken part in such a pool, it involves picking one team each week that you think will win. If your team wins in Week 1, you move on to make a pick in Week 2. If your team wins in Week 2, you can make a pick in Week 3, and so on. The catch is that you cannot pick the same team twice. You could use a heavy favorite in Week 1, but you would not have access to them later in the year if you're alive in the closing weeks. If you pick a team that loses, you are out of the pool for the remainder of the season.</p>
+
+          <p class="subtitle">Entry Fees and Payouts</p>
+          <p>The cost for this confidence pool is {{ poolCost }} per person for the whole season. The survivor pool mini game is
+            {{ survivorCost }} for the whole season which you can sign up for in the pool after registering. Please pay by your total entry fee by the end of the third week of the season (before the end of {{ payByDate }}). Payouts will be announced after that and will be based
```

Comment:
@balexander82 Probably should update this to say "during or after registering", now that we changed it

Reply:
@balexander82 Looks like a typo here, should be "Please pay your total entry fee" (there's an extra `by` in there)